projects/novo-elements/src/elements/data-table/data-table.pipes.ts

```diff
@@ -46,7 +46,8 @@ export class DateTableDateTimeRendererPipe<T> implements PipeTransform {
   constructor(private labels: NovoLabelService) {}
   transform(value: any, column: IDataTableColumn<T>): string {
     if (!Helpers.isEmpty(value)) {
-      return column.format ? value : this.labels.formatDate(interpolateCell<T>(value, column));
+      let val = interpolateCell<T>(value, column);
```

Comment:
Can you use const instead of let here?

Reply:
That's a possibility. However, this is for a hotfix, and is a simple reversion to what was there before. As such, I'd rather play it safe with what I know works and move forward quickly. A follow-on ticket will be set up to address concerns such as these.